config/kube.py

```diff
@@ -82,9 +65,7 @@ def server():
 @pass_db
 def ip(db, vip):
     """Specify a kubernetes cluster VIP"""
-    if vip and not is_valid_ip4_addr(vip) and not is_valid_ip6_addr(vip):
```

Comment:
Can we assume the URL is valid?

Reply:
The only validation is upon join. At the point of configuration, the URL need not be available. SONiC will periodically try until join.